package.json

```diff
@@ -40,6 +42,13 @@
     "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/performance/phpunit.xml.dist --verbose'",
     "pretest-php-multisite": "wp-env run composer 'install --no-interaction'",
     "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/performance/tests/multisite.xml --verbose'",
-    "wp-env": "wp-env"
+    "wp-env": "wp-env",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.php": [
+      "phpcbf --standard=phpcs.xml.dist",
+      "phpcs --standard=phpcs.xml.dist"
```

Comment:
I think this one should be `npm run lint-php` instead

Same as the above ideally the command should run from within the docker image in order to ensure it uses the right version from `composer.lock` file.

Reply:
@kirtangajjar we probably do not want to run `phpcbf` on staged files - that would make changes in the track. And potentially unwanted changes that we may not be able to get around. I would say just running `phpcs` or `npm run lint-php` would suffice here.

Reply:
Fixed.